tests/checkpointing/test_model_checkpoint.py

```diff
@@ -544,3 +546,79 @@ def validation_step(self, batch, batch_idx):
 
     trainer.fit(model)
     assert os.listdir(tmpdir) == ['epoch=00.ckpt']
+
+    def get_last_checkpoint():
```

Comment:
should we add a function like that somewhere to the model, that it get's (and maybe also loads) the last checkpoint from a given directory? Could be great for ux...

Reply:
Hey @justusschock, I think it is already implemented, but I don't know enough this part of the code :)